Resources/Prototypes/_White/Entities/Mobs/NPCs/headcrab.yml

```
@@ -0,0 +1,157 @@
+    - BaseMob
+    - MobDamageable
+    - MobCombat
+    - MobAtmosStandard
+    - MobBloodstream
+    - MobFlammable
+  id: BaseMobHeadcrab
+  abstract: true
+  name: headcrab
+  description: It looks like something parasitic.
+  components:
+  - type: GhostTakeoverAvailable
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)